src/CachePool.php

```diff
@@ -191,8 +191,7 @@ public function deleteMultiple($keys)
     public function has($key)
     {
         $default = $this->defaultValue;
-        $prefix = $this->getOptionNamePrefix();
-        $value = $this->getOption("{$prefix}{$key}", $default);
+        $value = $this->get($key, $default);
```

Comment:
As far as I remember, there was some reason why I didn't just do this in the beginning. But I don't remember what it was... Could you kindly consider this again, in case you think of something?

Reply:
I don't know why you wouldn't though. Here are some facts:

1. `get_transient()` **must** be called in order to correctly verify if a transient exists, because its this function that deletes expired transients
2. To check for false negatives, `get_option()` should be used.

The `CachePool::get()` method does both of these, in that order. It's perfect for the job.

Reply:
Mkay.